src/RDF/NodeUri/ResourceFactory/ResourceFactoryWithBlankNodes.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\RDF\NodeUri\ResourceFactory;
+
+use EasyRdf\Resource;
+
+/*
+ * @todo The current implementation, can be removed after https://jira.publiq.be/browse/III-6450 has been deployed and verified on production.
```

Comment:
Maybe remove this TODO and create a ticket for it?

Reply:
https://jira.publiq.be/browse/III-6498